### PR TITLE
docs(admob): document serverSideVerificationOptions

### LINKED ISSUE #4095: Add documentation for AdMob ServerSideVerification
Wow thanks for doing this @iAmGhost :)

Re the docs for `rnfirebase.io/reference/admob/requestoptions` this is generated so only need to update the types (which I think you have already).

Adding something to `rnfirebase.io/admob/displaying-ads#rewarded-ads` would be good though :)

_Originally posted by @Salakar in https://github.com/invertase/react-native-firebase/pull/4083#issuecomment-674036200_

### PR DESCRIPTION
**Pull request by iAmGhost**
### Description

Updated AdMob document for explaining new feature implemented in #4083.

Writing document were harder than writing code for me, I tried and did basic spell check, but please feel free to improve it.

### Related issues

Fixes #4095

### Checklist

- I read the [Contributor Guide](../CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
  - [x] Yes
- My change supports the following platforms;
  - [x] `Android`
  - [x] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `packages/\*\*/e2e`
  - [ ] `jest` tests added or updated in `packages/\*\*/__tests__`
- [ ] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [x] No

### Test Plan

I was unable to figure out how to build html pages myself but it was okay on [Gist](https://gist.github.com/iAmGhost/87b48a064bc3bc16d575768a2dd7d9e4).

---

Think `react-native-firebase` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter

### CODE REVIEW

File: docs/admob/displaying-ads.md [hunk 1]
```diff
@@ -195,6 +195,27 @@ To learn more, view the [`RewardedAdEventType`](/reference/admob/rewardedadevent
 
 If needed, you can reuse the existing instance of the `RewardedAd` class to load more adverts and show them when required.
 
+While the `EARNED_REWARD` event only occurs on the client, Server Side Verification (or SSV) can be used for confirming a user completed an advert action. For this, you have to specify Server Side Verification callback URL in dashboard.
```
Comment (Salakar) [inline docs/admob/displaying-ads.md:198]:
```suggestion
While the `EARNED_REWARD` event only occurs on the client, Server Side Verification (or SSV) can be used for confirming a user completed an advert action. For this, you have to specify the Server Side Verification callback URL in your Ads dashboard.
```

**Comment by mikehardy** [comment]
I find writing documentation to be very difficult also - I don't think many people appreciate how hard good documentation really is. Thanks for this and the original feature of course, really appreciated!

File: docs/admob/displaying-ads.md [hunk 2]
```diff
@@ -195,6 +195,27 @@ To learn more, view the [`RewardedAdEventType`](/reference/admob/rewardedadevent
 
 If needed, you can reuse the existing instance of the `RewardedAd` class to load more adverts and show them when required.
 
+While the `EARNED_REWARD` event only occurs on the client, Server Side Verification (or SSV) can be used for confirming a user completed an advert action. For this, you have to specify the Server Side Verification callback URL in your Ads dashboard.
+
+You can also customize parameters when your SSV callback is called by setting `serverSideVerificationOptions` in [`RequestOptions`](/reference/admob/requestoptions) parameter.
```
Comment (Salakar) [inline docs/admob/displaying-ads.md:200]:
```suggestion
You can customize SSV parameters when your SSV callback is called by setting the `serverSideVerificationOptions` field in your [`RequestOptions`](/reference/admob/requestoptions) parameter.
```

**Comment by Salakar** [comment]
Thank you :) tweaked the wording slightly but this was great 🎉

**Comment by iAmGhost** [comment]
> 
> 
> Thank you :) tweaked the wording slightly but this was great 🎉

I felt original one is a bit goofy since I'm not great at writing documents in English, Thank you very much for adjusting them!

**Pull request merged**